source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Jenkins
+description: How to configure continuous integration for a Drupal site on Pantheon.
+tags: [siteintegrations]
+type: guide
+permalink: docs/:basename/
+draft: true
+date: 5/30/2017
+
+---
+
+## Continuous Integration with Jenkins
+
+[Jenkins](https://jenkins.io) is an open source Continuous Integration server which can be used to build, test, and deploy code on any Drupal and WordPress website on Pantheon. Unlike services such as [CircleCI](https://circleci.com/), it is a Java application installed and run on a server, and requires regular upkeep and maintenance. However, it is very customizable and can even run non-CI related tasks, such as calling periodic cron jobs and other site-related tasks.
+
+## What you will build
+
+This guide will cover how to configure an existing Jenkins server to work with sites on the Pantheon platform. Because we want to submit pull requests and take advantage of existing GitHub and Jenkins integration, we will configure GitHub as the main code repository. When we push code to the GitHub repo, it will trigger Jenkins to test our code changes on a multidev environment. Jenkins will let us know if changes have passed and are ready to merge into the master branch on Pantheon. When we merge, Jenkins will run tests again against Pantheon's master branch.
```

Comment:
Phrasing nit: `let us know if changes have passed` + `When we merge, Jenkins will run tests` sounds a bit ambiguous vis-a-vis when tests are running. Rephrase to make it clear that both the PR and the master branch are tested.